<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>800</width>
				<height>600</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QVBoxLayout" name="verticalLayout">
				<item>
					<layout class="QHBoxLayout">
						<item>
							<spacer name="horizontalSpacer">
								<property name="sizeType">
									<enum>QSizePolicy::Expanding</enum>
								</property>
								<property name="orientation">
									<enum>Qt::Horizontal</enum>
								</property>
							</spacer>
						</item>
						<item>
							<widget class="QPushButton" name="btn_face_recognition">
								<property name="text">
									<string>Face Recognition</string>
								</property>
							</widget>
						</item>
						<item>
							<spacer name="horizontalSpacer_2">
								<property name="sizeType">
									<enum>QSizePolicy::Expanding</enum>
								</property>
								<property name="orientation">
									<enum>Qt::Horizontal</enum>
								</property>
							</spacer>
						</item>
					</layout>
				</item>
				<item>
					<widget class="QPushButton" name="btn_add_camera">
						<property name="text">
							<string>Add Camera</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="btn_modify_camera">
						<property name="text">
							<string>Modify Camera</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="btn_display_camera">
						<property name="text">
							<string>Display Cameras</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="btn_delete_camera">
						<property name="text">
							<string>Delete Camera</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="btn_add_person">
						<property name="text">
							<string>Add Person</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="btn_modify_person">
						<property name="text">
							<string>Modify Person</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="btn_display_person">
						<property name="text">
							<string>Display Persons</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="btn_delete_person">
						<property name="text">
							<string>Delete Person</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="btn_exit">
						<property name="text">
							<string>Exit</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QListWidget" name="personListWidget"/>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>
